app/components/shared/Icons/CloseIcon.tsx

```diff
@@ -0,0 +1,28 @@
+const CloseIcon = () => {
+  return (
+    <svg
+      fill="none"
+      height="16"
+      viewBox="0 0 16 16"
+      width="16"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M12 4L4 12"
+        stroke="#BDBCDB"
+        stroke-linecap="round"
```

Comment:
This usually throws some warnings/errors on browser consoles. I think the right property is strokeLinecap, strokeLinejoin, etc. 

If I'm wrong feel free to merge!